Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,142 @@
+
+        - ![Save](../../Assets/Icons/Save.png "[Save]") save  
+            Click this entry to save the current view settings in the selected view. This menu entry is only displayed if you have selected a view.
+
+            > [Info] When the settings of a view have been changed, an asterisk is displayed next to the view name. The asterisk is hidden as soon as the changes have been saved.
+
+        - ![Delete](../../Assets/Icons/Trash01.png "[Delete]") delete  
+            Click this entry to delete the selected view. A confirmation window to confirm the deletion is displayed. This menu entry is only displayed if you have selected a view.
+ 
+
+- *Connections*   
+    Click the drop-down list to select a connection. The following options are available:
+    - **All connections**  
+        Select this option to display authorization transactions for all configured connections. 
+    - **Connection name**  
+        Select this option to display authorization transactions for the selected connection. When a specific connection has been selected, the **All connections** option is no longer displayed. Click the selected connection again to deselect it.  
+- [x]     
+    Select the checkbox to the left of one list entry to display the editing toolbar for this single entry.   
+    If you click the checkbox in the header, all entries in the list are selected. The editing toolbar shows all actions you can perform on several entries.
+- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   
+    Click this button to display the search bar and search for entries.
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+    Click this button to update the entries.
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+    Click this button to display the columns bar and to customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.   
+    > [Info] It is recommended to add the following columns to your view to get the required information and to better filter and sort the entries: Status ID, Status information (if applicable). 
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+    Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
+
+The list displays all authorization transactions. Depending on the payment service provider, the displayed columns vary. All fields are read-only. The following standard columns are available.
+<!----Hallo Stefan, hier sollten alle Standard-Felder mit Erklärungen stehen, ist die Liste vollständig?--->
+- *Charges intended use*   
+   Purpose of the fee   
+- *Original currency*   
+   Currency in which the product(s) were paid   
+- *Original currency amount*   
+   Invoice amount in the original currency   
+
```

Comment:
Für die Standardspalten und Info kannst du generell in deinem Sandbox gucken, z.B.:
![image](https://github.com/Actindo-AG/Documentation/assets/99268392/d17c86cf-45a8-4bda-91b3-142ca8805f80)
Es sieht aber sehr ähnlich aus wie bei NoE. Frag mal am Besten Stefan, aber in der Regel, beschreiben wir bis ID, Created/Modified At + die Angezeigte, und alle, die ein Attribute sind beschreiben wir unter generell mit einem Eintrag als "Attribute name"... (z.B. aus Fulfillment, Omni-Channel)
![image](https://github.com/Actindo-AG/Documentation/assets/99268392/106d589c-28f2-4718-abeb-39feeb12ba31)